blueocean-core-js/package.json

```diff
@@ -76,15 +76,10 @@
     "gulp-babel": "6.1.2",
     "gulp-copy": "0.0.2",
     "gulp-eslint": "3.0.1",
-    "gulp-jest": "1.0.0",
     "gulp-less": "3.1.0",
     "gulp-rename": "1.2.2",
     "gulp-sourcemaps": "1.6.0",
-    "jest": "19.0.2",
```

Comment:
I am tending to include those "second hand" libraries lately since not having them explicitly declared can lead to weird versions.

Reply:
yes this would explain a lot ;)

Reply:
Yep, good tip, I may end up adding `jest` back in once I sort out some of these other problems.